szperacz/files.py

```diff
@@ -31,7 +31,6 @@ def _get_datetime_data(self, data, image):
         except AttributeError:
             data['creation_time'] = None
 
-
```

Comment:
Changes like this shall not be a part of commit/pull request. It is pure, meaningless noise and is not related to solution of the task.

Please revert this.